_posts/2021-03-06-newpipe-0.20.11-released.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "NewPipe 0.20.11 released: Hotfixing YouTube"
+short: "NewPipe 0.20.10 released"
+date: 2021-03-06 11:00:00 + 1:00
+categories: [pinned, release]
+author: Opusforlife & Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Welcome to the first hotfix release of 2021 - let us hope it will be the last one. This release fixes receiving some YouTube content, an app crash and brings you a new translation.
```

Comment:
```suggestion
Welcome to the first hotfix release of 2021 - let us hope it will be the last one. This release fixes SoundCloud, receiving some YouTube content, an app crash and brings you a new translation.
```